website/app/styles/pages/home.scss

```diff
@@ -120,7 +120,7 @@ body.application.index {
   }
 
   p {
-    @include doc-font-family("gilmer");
+    @include doc-font-family("hashicorp-sans");
```

Comment:
In looking at the hashicorp.com pages and from what I recall of the rebrand slides, the new font is for headings and logos, not body copy. Can this be updated to use the `sans` font-family instead? It probably should've been this way from the beginning and was just missed.